### PR TITLE
Popup window implementation

### PR DESCRIPTION
**Pull request by BANCUNGUYE66**
added javascript to display cards and popup window.

### CODE REVIEW

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
Hi @BANCUNGUYE66 :wave: , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

invalid

## Invalid Code Review Request

You have submitted a pull request with no history of pair-programming activity with your coding partner/s. Kindly ensure you collaborate with your partner/s as it is a requirement. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -40,6 +68,220 @@ portfolio website
...
+
+This project is [MIT](./LICENSE.md) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+  </ul>
+
+</details>
+
```
Comment (emmanuellekamwa) [inline README.md]: - [x] Your README file contain some duplicate headings. Please cross-check it and delete all that should be deleted.

Comment (emmanuellekamwa) [inline README.md]: - [x] Your README file contain some duplicate headings. Please cross-check it and delete all that should be deleted.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ⛔🛑**
Hello @BANCUNGUYE66 👋
Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there.

## Highlights
- Your linters are passing all the checks.

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @BANCUNGUYE66 , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Collaborative work :+1: Commits from all team members
:heavy_check_mark: Popup works exactly as expected
:heavy_check_mark:  Implemented all suggested changes
:heavy_check_mark: Descriptive Readme file and PR

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**